README.md

```diff
@@ -61,30 +60,115 @@ nix build github:determinatesystems/harmonic
 
 Then copy the `result/bin/harmonic` to the machine you wish to run it on.
 
-## Running
+## Installing
 
-Harmonic must be run as `root`, as it needs to alter the system and cannot elevate privileges without significant complexity.
```

Comment:
I'd actually rather keep this, but maybe reword to say "this tool requires root, but we can elevate ourselves with `sudo` if you happen to forget" -- users who only have e.g. `doas` or `please` available as their elevation tools won't be able to rely on the automatic elevation (since we hardcode `sudo`, but also they may have esoteric rules that prevent a potential `doas` auto-run from working).